<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Browser</class>
 <widget class="QMainWindow" name="Browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/logo2.bmp</normaloff>:/img/logo2.bmp</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.488, y1:0, x2:0.497, y2:1,
stop:0 rgba(181, 177, 177, 255), 
stop:1 rgba(69, 64, 64, 255));
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background-color: qlineargradient(spread:pad, x1:0.488, y1:0, x2:0.497, y2:1,
stop:0 rgba(170, 164,164, 255), 
stop:1 rgba(69, 64, 64, 255));
border-bottom:1px solid rgb(79,101,125);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>33</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>33</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

border-radius: 5px;
background-image: url(button_left2.bmp);

}

QPushButton:hover{
 border: none;
 background-image: url(left.bmp);
}

QPushButton:hover:pressed {
 border: none;
 background-image: url(left.bmp);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>33</width>
           <height>23</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>33</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>33</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

border-radius: 5px;
 background-image: url(batton_right2.bmp);
}

QPushButton:hover{
 border: none;
 background-image: url(right.bmp);
}

QPushButton:hover:pressed {
 border: none;
 background-image: url(right.bmp);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>33</width>
           <height>23</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>33</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>33</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

border-radius: 5px;
 background-image: url(update4.bmp);
}

QPushButton:hover{
 border: none;
 background-image: url(update.bmp);
}

QPushButton:hover:pressed {
 border: none;
 background-image: url(update.bmp);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>33</width>
           <height>23</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>23</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
background-color: rgb(255, 255, 255);
border: 1px solid #707070;
border-radius: 3px;
}
</string>
         </property>
         <property name="text">
          <string>http://www.bash.org.ru</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

border-radius: 5px;
 background-image: url(x.bmp);
}

QPushButton:hover{
 border: none;
 background-image: url(x2.bmp);
}

QPushButton:hover:pressed {
 border: none;
 background-image: url(x2.bmp);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWebView" name="webView"/>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>395</width>
        <height>15</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
border-bottom:1px solid rgb(0,0,0);
border-top:1px solid rgb(0,0,0);
border-left:1px solid rgb(0,0,0);
border-right:1px solid rgb(0,0,0);
text-align:center;
background-color: rgba(0, 0, 0, 0);
}

QProgressBar::chunk {
background-color: qlineargradient(spread:pad, x1:0.488, y1:0,
x2:0.497, y2:1, stop:0 #85B3E8, stop:1 #3067a5);
width: 9px;
margin: 1px;
margin-right:0px;
padding:1px;
border:1px solid black;
}
</string>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuXcb">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_8"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="separator"/>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
    <addaction name="action_15"/>
    <addaction name="separator"/>
    <addaction name="action_17"/>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_20"/>
    <addaction name="action_21"/>
    <addaction name="separator"/>
    <addaction name="action_23"/>
    <addaction name="action_24"/>
    <addaction name="action_25"/>
    <addaction name="action_26"/>
    <addaction name="separator"/>
    <addaction name="action_28"/>
    <addaction name="separator"/>
    <addaction name="action_31"/>
    <addaction name="separator"/>
    <addaction name="action_34"/>
    <addaction name="action_35"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид </string>
    </property>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>Панели инструментов</string>
     </property>
     <addaction name="action_37"/>
     <addaction name="action_38"/>
     <addaction name="action_39"/>
     <addaction name="action_40"/>
     <addaction name="action_41"/>
     <addaction name="action_42"/>
     <addaction name="action_43"/>
     <addaction name="separator"/>
     <addaction name="action_45"/>
    </widget>
    <widget class="QMenu" name="menu_8">
     <property name="title">
      <string>Масштаб</string>
     </property>
     <addaction name="action50"/>
     <addaction name="action80"/>
     <addaction name="action90"/>
     <addaction name="separator"/>
     <addaction name="action100"/>
     <addaction name="separator"/>
     <addaction name="action120"/>
     <addaction name="action150"/>
     <addaction name="action200"/>
     <addaction name="separator"/>
     <addaction name="action_100"/>
     <addaction name="action_101"/>
     <addaction name="action_50"/>
     <addaction name="action_51"/>
    </widget>
    <addaction name="menu_7"/>
    <addaction name="separator"/>
    <addaction name="menu_8"/>
    <addaction name="action_52"/>
    <addaction name="action_56"/>
    <addaction name="action_57"/>
    <addaction name="separator"/>
    <addaction name="action_59"/>
    <addaction name="separator"/>
    <addaction name="action_61"/>
    <addaction name="action_62"/>
    <addaction name="action_63"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Закладки</string>
    </property>
    <addaction name="action_64"/>
    <addaction name="action_65"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Виджеты </string>
    </property>
    <addaction name="action_67"/>
    <addaction name="action_68"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="action_69"/>
    <addaction name="action_70"/>
    <addaction name="action_71"/>
    <addaction name="action_72"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_73"/>
    <addaction name="separator"/>
    <addaction name="action_75"/>
   </widget>
   <addaction name="menuXcb"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Создать вкладку</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Создать окно</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Открвть</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Добавить вкладку</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Сеансы</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Импорт и экспорт</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Параметры печати</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Просмотр печати</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Синхронизировать</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>Работать автономно</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>Вернуть</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>Вырезать</string>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="action_25">
   <property name="text">
    <string>Копировать в заметки</string>
   </property>
  </action>
  <action name="action_26">
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="action_28">
   <property name="text">
    <string>Выбрать все</string>
   </property>
  </action>
  <action name="action_31">
   <property name="text">
    <string>Проверить орфографию</string>
   </property>
  </action>
  <action name="action_34">
   <property name="text">
    <string>Найти</string>
   </property>
  </action>
  <action name="action_35">
   <property name="text">
    <string>Найти следующее</string>
   </property>
  </action>
  <action name="action_37">
   <property name="text">
    <string>Панели</string>
   </property>
  </action>
  <action name="action_38">
   <property name="text">
    <string>Главная панель</string>
   </property>
  </action>
  <action name="action_39">
   <property name="text">
    <string>Личная панель</string>
   </property>
  </action>
  <action name="action_40">
   <property name="text">
    <string>Панель вкладок</string>
   </property>
  </action>
  <action name="action_41">
   <property name="text">
    <string>Панель состояния</string>
   </property>
  </action>
  <action name="action_42">
   <property name="text">
    <string>Панель адреса</string>
   </property>
  </action>
  <action name="action_43">
   <property name="text">
    <string>Панель навигации</string>
   </property>
  </action>
  <action name="action_45">
   <property name="text">
    <string>Настроить...</string>
   </property>
  </action>
  <action name="action50">
   <property name="text">
    <string>50%</string>
   </property>
  </action>
  <action name="action80">
   <property name="text">
    <string>80%</string>
   </property>
  </action>
  <action name="action90">
   <property name="text">
    <string>90%</string>
   </property>
  </action>
  <action name="action100">
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="action120">
   <property name="text">
    <string>120%</string>
   </property>
  </action>
  <action name="action150">
   <property name="text">
    <string>150%</string>
   </property>
  </action>
  <action name="action200">
   <property name="text">
    <string>200%</string>
   </property>
  </action>
  <action name="action_100">
   <property name="text">
    <string>-100%</string>
   </property>
  </action>
  <action name="action_101">
   <property name="text">
    <string>+100%</string>
   </property>
  </action>
  <action name="action_50">
   <property name="text">
    <string>-10%</string>
   </property>
  </action>
  <action name="action_51">
   <property name="text">
    <string>+10%</string>
   </property>
  </action>
  <action name="action_52">
   <property name="text">
    <string>Изображения</string>
   </property>
  </action>
  <action name="action_56">
   <property name="text">
    <string>Стиль</string>
   </property>
  </action>
  <action name="action_57">
   <property name="text">
    <string>Кодировка</string>
   </property>
  </action>
  <action name="action_59">
   <property name="text">
    <string>Исходный код</string>
   </property>
  </action>
  <action name="action_61">
   <property name="text">
    <string>Маленький экран</string>
   </property>
  </action>
  <action name="action_62">
   <property name="text">
    <string>Подогнать по ширине</string>
   </property>
  </action>
  <action name="action_63">
   <property name="text">
    <string>Во весь экран</string>
   </property>
  </action>
  <action name="action_64">
   <property name="text">
    <string>Создать закладку страницы...</string>
   </property>
  </action>
  <action name="action_65">
   <property name="text">
    <string>Управление закладками...</string>
   </property>
  </action>
  <action name="action_67">
   <property name="text">
    <string>Добавить виджеты</string>
   </property>
  </action>
  <action name="action_68">
   <property name="text">
    <string>Управление виджетами</string>
   </property>
  </action>
  <action name="action_69">
   <property name="text">
    <string>Заметки</string>
   </property>
  </action>
  <action name="action_70">
   <property name="text">
    <string>Загрузки</string>
   </property>
  </action>
  <action name="action_71">
   <property name="text">
    <string>Истории </string>
   </property>
  </action>
  <action name="action_72">
   <property name="text">
    <string>Ссылки</string>
   </property>
  </action>
  <action name="action_73">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="action_75">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends></extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webView</sender>
   <signal>loadProgress(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
